card.py

```diff
@@ -1,8 +1,26 @@
 class Card:
+    face_value_mapping = {
+        1: 'A',
+        11: 'J',
+        12: 'Q',
+        13: 'K',
+    }
+
     def __init__(self, v, s):
-        self.value = v
         self.suit = s
+        self.value = v
+        self.value_name = self.face_value_mapping.get(self.value, self.value)
```

Comment:
This is fine for now. Since `value_name` is not something that can change after more code is running, do it this way will work. There will be a better example of using `@property` later in `player.py` when we get there.